### PR TITLE
EVG-5530 redo UI for triggers

### PR DESCRIPTION
**Pull request by john-m-liu**
screen shot 2018-12-12 at 5 22 43 pm

### CODE REVIEW

**Comment by john-m-liu** [comment]
The ticket mentions adding an alias which I will do in a different ticket

File: trigger/process.go [hunk 1]
```diff
@@ -159,6 +160,9 @@ projectLoop:
 			if trigger.Status != "" && trigger.Status != t.Status {
 				continue
 			}
+			if trigger.DateCutoff != nil && time.Now().Add(-24*time.Duration(*trigger.DateCutoff)*time.Hour).After(t.IngestTime) {
```
Comment (tychoish) [inline trigger/process.go]: I think `time.duration(*trigger.DateCutoff)` is can omit the deference?

File: public/static/partials/project_trigger_modal.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Task regex:</label>
+                            <input type="text" ng-model="trigger.task_regex" placeholder="Tasks to run">
+                        </md-input-container>
+                    </section>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Command:</label>
+                            <input type="text" ng-model="trigger.command" placeholder="Command that creates task json file">
```
Comment (tychoish) [inline public/static/partials/project_trigger_modal.html]: I feel like it's not clear that you should put json/yaml here? or is this supposed to be a shell command?

File: public/static/partials/project_trigger_modal.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <div ng-bind="modalTitle()"></div>
+                </md-card-title>
+                <md-card-content>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Project:</label>
+                            <input type="text" ng-model="trigger.project" ng-required="true">
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Date cutoff:</label>
```
Comment (tychoish) [inline public/static/partials/project_trigger_modal.html]: I feel like we should use a different term than date cutoff? let's think/poll some people on this.

File: public/static/partials/project_trigger_modal.html (hunk 2 above)
Comment (tychoish) [reply public/static/partials/project_trigger_modal.html]: nit: put the word shell in the ghost text.

**Comment by tychoish** [comment]
modulo nit

**Pull request merged**